app/models/host.rb

```diff
@@ -1881,5 +1881,16 @@ def read_only_storages
     storages.where(:host_storages => {:read_only => true})
   end
 
+  def archived?
+    ems_id.nil?
```

Comment:
I'm always a bit cautious on checking `_id` fields, because they can be false positives.
Would `ext_management_system.nil?` cause a performance impact or is this already loaded?